terraform/meta/main.tf

```diff
@@ -122,7 +122,38 @@ data "aws_caller_identity" "current" {}
 module "github_actions_access" {
   source = "../modules/github_actions_access"
 
-  application_repo = "communitiesuk/submit-social-housing-lettings-and-sales-data"
-  ecr_arn          = module.ecr.repository_arn
-  meta_account_id  = data.aws_caller_identity.current.account_id
+  meta_account_id = data.aws_caller_identity.current.account_id
+  repositories = {
+    application = {
+      name = "communitiesuk/submit-social-housing-lettings-and-sales-data",
+      policies = [
+        { key = "push_ecr_images", arn = module.ecr.push_images_policy_arn },
+        { key = "access_non_prod_state", arn = module.non_prod_backend.state_access_policy_arn }
+      ]
+    },
+    infrastructure = {
+      name     = "communitiesuk/submit-social-housing-lettings-and-sales-data-infrastructure"
+      policies = []
+    }
+  }
+}
+
+moved {

```

Comment:
N.B. These moved blocks are left in for the review for clarity, but I'll remove them afterwards (they are useful for seeing what's going on, but aren't needed once the terraform has been run)